src/test/java/com/connexta/store/StatusServiceTests.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.isA;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.connexta.poller.service.StatusResponse;
+import com.connexta.poller.service.StatusService;
+import com.connexta.store.config.AmazonS3Configuration;
+import com.connexta.store.config.S3StorageConfiguration;
+import com.connexta.store.controllers.StoreController;
+import com.dyngr.exception.PollerStoppedException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import javax.inject.Inject;
+import okhttp3.HttpUrl;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+
+@SpringBootTest
```

Comment:
❗️`@SpringBootTest` should only be used if you need to start up the whole application. Since this test is only testing the StatusService, this should be a unit test. You shouldn't have to mock things that aren't used in StatusService.

Reply:
I'll switch it over to unit test. It will take a little time. I'll work on it when I get back from my PB.